### PR TITLE
added Works Section and work cards in this commit

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Add these Changes and Updates to the Project:

1. Works Section
2. Work Cards
3. Buttons

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,10 +58,269 @@ <h2 class="headline-text1">I’m Wahab.
...
+                        </button>
+                    </div>
+
+
+                </div>
+
+            </div>
+
+            <div class="Card-works">
+                <div class="snapshoot-container">
```
Comment (hiromataba) [inline index.html:154]: - [ ] Please check the original design as suggested in the below [video](https://youtu.be/jiIed23OPMA) recorded for goo understanding 😉

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi @Abdulwhbhussain     :wave: :wave: 

almost-there

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Your PR has a proper title and description  :+1: 
- Your linters are passing :heavy_check_mark:  
- Profesional README file :rocket: 

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -84,3 +85,969 @@
...
+
+  /* Works */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
```
[the remark this answers is not in the document]
Comment (Abdulwhbhussain) [reply style.css:102]: ok! i implemented it.

File: index.html [hunk 3]
```diff
@@ -59,10 +58,269 @@ <h2 class="headline-text1">I’m Wahab.
...
+                        </div>
+                    </div>
+
+                    <p class="text-after-Frame2">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+                    <div class="tags">
+                        <div class="tag-html"><p class="tag-text-html">HTML</p></div>
+                        <div class="tag-css"><p class="tag-text-css">CSS</p></div>
+                        <div class="tag-js"><p class="tag-text-js">JavaScript</p></div>
+                    </div>
```
[the remark this answers is not in the document]
Comment (Abdulwhbhussain) [reply index.html:97]: ok.

**Review by LionRouge1 — APPROVED** [review]
Hi @Abdulwhbhussain , 

# STATUS: APPROVED! 🥇 💯 
- Your portfolio looks almost identic to Figma's design :+1: 
done

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**